DICOMwebProxy/src/main/java/online/kheops/proxy/wadors/WadoRsResource.java

```diff
@@ -75,7 +75,7 @@ public Response wadoInstance(@HeaderParam(AUTHORIZATION) String authorizationHea
     }
 
     @GET
-    @Path("/password/dicomweb/studies/{studyInstanceUID:([0-9]+[.])*[0-9]+}/series/{seriesInstanceUID:([0-9]+[.])*[0-9]+}/instances/{sopInstanceUID:([0-9]+[.])*[0-9]+}/bulkdata/{tag:([a-fA-F0-9]+)}")
+    @Path("/password/dicomweb/studies/{studyInstanceUID:([0-9]+[.])*[0-9]+}/series/{seriesInstanceUID:([0-9]+[.])*[0-9]+}/instances/{sopInstanceUID:([0-9]+[.])*[0-9]+}/bulkdata/{tag:([a-fA-F0-9/]+)}")
```

Comment:
Allow a / in the tag path so that, for example:
http://127.0.0.1/api/link/password/studies/1.3.6.1.4.1.53684.1.1.2.4037847388.7016.1636643714.1136392/series/2.25.731164836577152435888135968415415768727/instances/2.25.838838149706387630222213118949206286602/bulkdata/0040A730/4/0040A730/0/0040A730/2/0040A730/0/00700022
works.  This is the actual form used by dcm4chee for long graphic data values.